handlers/officersHandler.go

```diff
@@ -0,0 +1,32 @@
+package handlers
+
+import (
+	"encoding/json"
+	"github.com/companieshouse/chs-delta-api/models"
+	"net/http"
+)
+
+// OfficerHandler offers a handler by which to publish a message onto a kafka topic.
+type OfficerHandler struct {
+}
+
+// NewOfficerHandler returns a OfficerHandler.
+func NewOfficerHandler() *OfficerHandler {
+	return &OfficerHandler{
+	}
+}
+
+// ServeHTTP accepts an incoming officer request via a POST method and validates and forwards to the XXX service.
+func (kp *OfficerHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+
+	var officersData models.OfficerDelta
+
+	// If the decoding fails, it will most likely be due to bad data being submitted by the user.
```

Comment:
Should we set the Content-Type for the response ?

Reply:
Will look into adding this

Reply:
At the minute we're just returning the error from the json decoder which is text. When we implement the CH error resource maybe then is better time to set the content-type?